src/Nostr/Connection.ts

```diff
@@ -94,17 +99,11 @@ export default class Connection {
     OnOpen(e: Event) {
         this.ConnectTimeout = DefaultConnectTimeout;
         console.log(`[${this.Address}] Open!`);
-
-        // send pending
-        for (let p of this.Pending) {
-            this._SendJson(p);
-        }
-        this.Pending = [];
-
-        for (let [_, s] of this.Subscriptions) {
-            this._SendSubscription(s);
-        }
-        this._UpdateState();
+        setTimeout(() => {
```

Comment:
I did some tests and it REALLY doesn't like this timeout.

That's fine the subscriptions will just be re-requested on auth, which happens. I'll do some further testing.